h/static/scripts/test/root-thread-test.js

```diff
@@ -186,7 +187,7 @@ describe('rootThread', function () {
     ]);
   });
 
-  describe('when the thread filter query is set', function () {
+  describe('when no filter query is set', function () {
```

Comment:
The description of the tests here was incorrect previously. These tests check that when the user has _not_ entered a filter query, the sidebar only shows annotations that belong in the selected tab (Annotations, Page Notes or Orphans)